packages/google_maps_flutter/google_maps_flutter/test/map_creation_test.dart

```diff
@@ -29,8 +29,12 @@ void main() {
   ) async {
     // Inject two map widgets...
     await tester.pumpWidget(
+      // TODO(goderbauer): Make this const when that's available on stable.
```

Comment:
Could you put a version? That helps with knowing when to clean it up later; we often use language like "Make this const when this package requires Flutter 3.8 or later".

Reply:
Done using your suggested language.